rustls/src/verify.rs

```diff
@@ -212,6 +212,15 @@ impl AsRef<str> for DnsName {
     }
 }
 
+impl fmt::Debug for DnsName {
+    // Workaroud solution for ServerName debug formatting:
```

Comment:
Tiny nit: "workaroud" -> "workaround".

Reply:
> Tiny nit: "workaroud" -> "workaround".

making this from the GH GUI:

```suggestion
    // Workaround solution for ServerName debug formatting:
```